build.sbt

```diff
@@ -14,7 +14,8 @@ val doobieVersion = "1.0.0-RC2"
 val http4sVersion = "0.23.14"
 val http4sBlazeVersion = "0.23.12"
 val circeVersion = "0.14.2"
-val tsecVersion = "0.4.0"
+val password4jVersion = "1.6.0"
```

Comment:
At each hashing function usage `password4j` would grumble in logs, that `argon2` properties are not set. We could provide default properties, but in production environment they should be not default, but rather set in accordance with deployment machine... so perhaps it is good, that it complains in logs and everything should be left as it is?

Reply:
hm is it possible to configure `password4j` using normal parameters, that is in code? The way configuration is done should be external to the library, in our case we have the `.conf` file where configuration resides, but it would be best if we could just use whatever functionality we need by creating classes / calling methods with the values that we think are correct

Reply:
As agreed, properties would be hardcoded in code, since it is unlikely that they would be changed after deployment